src/main/java/com/epam/processor/DataProcessor.java

```diff
@@ -71,18 +108,25 @@ public RoadAccident getAccidentByIndex7(String index){
      * @return
      */
     public Multimap<String, String> getAccidentIdsGroupedByAuthority7(){
-        return null;
+
+    	Multimap<String, String> accidentIdGrpByAuthority = (Multimap) ArrayListMultimap.create();
+    	for(RoadAccident accident : roadAccidentList) {
+    		
+    		accidentIdGrpByAuthority.put(accident.getDistrictAuthority(), accident.getAccidentId());
+    		
+    	}
+    return accidentIdGrpByAuthority;
     }
 
 
     // Now let's do same tasks but now with streaming api
 
 
-
-    public RoadAccident getAccidentByIndex(String index){
-        return null;
-    }
-
+	public RoadAccident getAccidentByIndex(String index) {
+		return roadAccidentList.stream()
+				.filter(accident -> accident.getAccidentId().equals(index))
+				.findFirst().orElse(null);
```

Comment:
Without Else you will have - NoSuchElementException  and it's even better, than NULL